src/app/loginComponents/requests/requests.component.scss

```diff
@@ -0,0 +1,4 @@
+.removeOrg {
```

Comment:
What does it look like before this class, what does it looks like after?

Reply:
Before:
<img width="1219" alt="Screen Shot 2020-09-01 at 2 00 57 PM" src="https://user-images.githubusercontent.com/42463687/91905582-99848780-ec5b-11ea-9100-51c202822c2c.png">
After:
<img width="1218" alt="Screen Shot 2020-09-01 at 2 00 25 PM" src="https://user-images.githubusercontent.com/42463687/91905623-aacd9400-ec5b-11ea-8b3f-6a27fbdb6a56.png">

The biggest difference would be how the cursor looks when hovering over the link. The before image would have the cursor as a normal pointer, but the after image would have the cursor hand pointer which is exactly the same pointer as hovering over the "Accept" or "Reject" buttons.

Reply:
Just use a button instead of anchor, automatically black and hover